tests/gocase/unit/type/bitmap/bitmap_test.go

```diff
@@ -234,6 +234,14 @@ func TestBitmap(t *testing.T) {
 		}
 	})
 
+	t.Run("GH-1714", func(t *testing.T) {
```

Comment:
please also mention the boundary word.
In my opinion, mention #1714 seems to have no value, ie there is nothing in it, we can merge the details to the comment and commit comment